kubernetes/migration/versions.go

```diff
@@ -1056,3 +1066,43 @@ var proxyRemoveHttpsGoogleProtocol = func(o *corefile.Option) (*corefile.Option,
 	}
 	return o, nil
 }
+
+func breakForwardStubDomainsIntoServerBlocks(cf *corefile.Corefile) (*corefile.Corefile, error) {
```

Comment:
If I understand this correctly, the migration of `proxy` to `forward` takes place first and then this function breaks them up into separate stanzas. Neat!